cypress/integration/components/header.spec.js

```diff
@@ -0,0 +1,22 @@
+describe("Testa comportamento do componente Header", () => {
+    it("Testa se os itens do componente estão visíveis", () => {
+
```

Comment:
alternativa: dessa maneira você não precisa fazer cy.visit em todo teste, pois antes de executar cada um ele faz
```suggestion
    beforeEach(() => {
        cy.visit('/');
    }
    it("Testa se os itens do componente estão visíveis", () => {
```